client/src/components/v3/StatisticRow.tsx

```diff
@@ -0,0 +1,78 @@
+import { Sensor, useSensorData } from 'api/common/data';
+import React, { useEffect, useState } from 'react';
+import { HeartRateRT, PowerRT, ReedVelocityRT } from 'types/data';
+import Statistic from './Statistic';
+import styles from './StatisticRow.module.css';
+
+/**
+ * Statistic container using API for various stats.
+ *
+ * @returns Component
+ */
+export default function StatisticRow(): JSX.Element {
+    // Should only be positive, abs max of currSpeed
+    const [maxSpeed, setMaxSpeed] = useState<number | null>(null);
+
+    // TODO: Multiple sensor data
+    const currSpeed = useSensorData(3, Sensor.ReedVelocity, ReedVelocityRT);
+
+    // Update max speed whenever the speed is updated
+    useEffect(() => {
+        // Add new data point
+        if (currSpeed) setMaxSpeed(Math.max(currSpeed, Math.abs(maxSpeed ?? 0)));
```

Comment:
Can you please add a comment just describing this line or split it up into multiple lines.

Reply:
Is it mainly the null coalescing operator that's making it hard to understand? I would have expected that the nested maths operators wouldn't be too confusing to follow as it pretty much reads from left to right.

Reply:
Just want to know what sort of things I should be looking out for.

Reply:
f6ddcbd